### ISSUE TITLE
Unnecessary automatic charging stop added

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#447]
>Car Data: X90D, Charger: 48, Wheels: 20, Tires: 0, Pano: 0, 5 Seat: 1, Rear CC: 0, Bat Life: 100, Payload: 100, PF: 6, SA: 0, Temp: 20 
Device Info: iPhone; CPU iPhone OS 10_3_2 like Mac OS X 
Settings: Reserve SoC:20, Default SoC: 90, Region: 0, Units: 0 
Version: 1.2.0 (37) 
IAP: Pro:1, Sub:1 
Location: Unknown 
Trip: Oxford Suites Portland - Jantzen Beach, [1l60p8sw41jg2wyk3q0ndw]

This trip shows a charging stop in Centralia with no added SoC. The arrival and departure SoC are both 57%.

**Comment by EVGrokker** [comment]
Here's another trip showing the same behavior, adding an unnecessary charging stop:

>Car Data: X90D, Charger: 48, Wheels: 20, Tires: 0, Pano: 0, 5 Seat: 1, Rear CC: 0, Bat Life: 100, Payload: 100, PF: 0, SA: 0, Temp: 20 
Device Info: iPhone; CPU iPhone OS 10_3_2 like Mac OS X 
Settings: Reserve SoC:20, Default SoC: 90, Region: 0, Units: 0 
Version: 1.2.0 (40) 
IAP: Pro:1, Sub:1 
Location: Unknown 
Trip: Fess Parker Inn, [w6ccxm7jsjpq31fhli2wmr]

I created a New Trip (Seattle - Fess Parker [HPWC]). I was going to do my normal pin and segment to chop it up, but I noticed that EVTO calls for a stop at Centralia with no charging (62>62), and the pin is colored in gray. What does a gray pin mean in this context?

![graypin](https://user-images.githubusercontent.com/26678403/27137971-8f35e1e8-50dc-11e7-8b9e-87b9a1fea94a.PNG)
